tests/app/notifications/test_validators.py

```diff
@@ -620,8 +620,9 @@ def test_check_service_sms_sender_id_where_sms_sender_is_not_found(sample_servic
     assert e.value.message == "sms_sender_id {} does not exist in database for service id {}".format(fake_uuid, sample_service.id)
 
 
+@pytest.mark.skip(reason="Disable temporarily to test logging")
 def test_check_service_over_bounce_rate_critical(mocker, fake_uuid):
-    mocker.patch("app.bounce_rate_client.check_bounce_rate_status", return_value=BounceRateStatus.CRITICAL.value)
+    mocker.patch("app.bounce_rate_client.check_bounce_rate_status", return_value="critical")
```

Comment:
If this change is important you may need to make a note to make it again once you revert this PR!